### DISCUSSION TITLE
1020 Error?

### DISCUSSION THREAD

**Discussion by freinkelhouse** [Robbbbbbbbb/OpenEFT#9] [Q&A] [answered]
I get this error:

Fingerprint cards cannot be validated. Please retry or submit hard copy fingerprint cards.

Additional Error Information:

1. Code: 1020 | Description: Missing mandatory Field 'T2_WGT' 2.29 IDC:0 | Additional Info:
2. 

Ive tried a few cards and scans. Not having much luck.  My 22nd try got this message. Anyone have any ideas? All fields filled except SSN!

**Comment by freinkelhouse** [comment]
Was confirmed that all alignment and fields were incorrectly mapping.

**Comment by Robbbbbbbbb** [comment, marked as answer]
Make sure your prints are legible and not smudged. If your ridges are not clearly defined (eg: too light or smudged) the file isn't producing accurate data and will fail validation.
